### ISSUE TITLE
different loss functions for loss_xy and loss_wh

### ISSUE THREAD

**Issue by love2learn-7179** [Tianxiaomo/pytorch-YOLOv4#71]
May I ask why you use F.binary_cross_entropy for loss_xy while you use F.mse_loss for loss_wh?
And, why did you divide by 2 for loss_wh? (you didn't divide by 2 for loss_xy)

Thank you for your help.

**Comment by Sooryakiran** [comment]
Hi. Same doubt here. Please let me know if you have any extra information regarding the same.
